debian/jitsi-meet-tokens.postinst

```diff
@@ -48,9 +48,9 @@ case "$1" in
         db_stop
 
         if [ -f "$PROSODY_HOST_CONFIG" ] ; then
-            # search for --plugin_paths, if this is not enabled this is the
+            # search for the token auth, if this is not enabled this is the
             # first time we install tokens package and needs a config change
-            if grep -q "\-\-plugin_paths" "$PROSODY_HOST_CONFIG"; then
+            if ! egrep -q '^\s*authentication\s*=\s*"token"' "$PROSODY_HOST_CONFIG"; then
```

Comment:
Is egrep available by default on all Debian/Ubuntu minimal images?

Reply:
grep and egrep are the same.

egrep = grep -E

Reply:
Ok, but is that available by default on minimal Debian distro? If not sure can you modify it to use grep, then?
Otherwise, that can brake installation on some distros...

Reply:
yes, available by default on minimal Debian distro

It comes with the `grep` package which is in the essential package list

```
apt show grep

Package: grep
Version: 3.3-1
Priority: required
Essential: yes
Section: utils
...
...
Description: GNU grep, egrep and fgrep
```

Reply:
Thanks for checking.